docs/upgrade-notes.asciidoc

```diff
@@ -0,0 +1,718 @@
+
+[discrete]
+.Removed `is_nan`, `is_finite`, and `is_infinite` functions from {esql}. (8.13)
+[%collapsible]
+====
+*Details* +
+These functions have been removed from {esql} queries as they are not supported. Errors would be thrown when trying to use them. For more information, refer to ({kibana-pull}174674[#174674]).
+====
+
+[discrete]
+.New SLO architecture. (8.12)
+[%collapsible]
+====
+*Details* +
+We introduce a breaking change in the SLO features that will break any SLOs created before 8.12. These SLOs have to be manually reset through an API until we provide a UI for it. The data aggregated over time (rollup) is still available in the sli v2 index, but won't be used for summary calculation when reset.
+
+The previous summary transforms summarizing every SLOs won't be used anymore and can be stopped and deleted:
+
+* slo-summary-occurrences-7d-rolling
+* slo-summary-occurrences-30d-rolling
+* slo-summary-occurrences-90d-rolling
+* slo-summary-occurrences-monthly-aligned
+* slo-summary-occurrences-weekly-aligned
+* slo-summary-timeslices-7d-rolling
+* slo-summary-timeslices-30d-rolling
+* slo-summary-timeslices-90d-rolling
+* slo-summary-timeslices-monthly-aligned
+* slo-summary-timeslices-weekly-aligned
+
+Be aware that when installing a new SLO (or after resetting an SLO), we install two transforms (one for the rollup data and one that summarize the rollup data). Do not delete the new `slo-summary-{slo_id}-{slo_revision}` transforms. For more information, refer to ({kibana-pull}172224[#172224]).
+====
+
+[discrete]
+.A new sub-feature privilege to control user access to the cases settings. (8.12)
+[%collapsible]
+====
+*Details* +
+Roles with at least a sub-feature privilege configured will not have access to the cases setting like they had previously. All roles without a sub-feature privilege configured will not be affected. For more information, refer to ({kibana-pull}170635[#170635]).
+====
+
+[discrete]
+.Improved config output validation for default output. (8.11)
+[%collapsible]
+====
+*Details* +
+Improve config output validation to not allow to defining multiple default outputs in {kib} configuration. For more information, refer to ({kibana-pull}167085[#167085]).
+====
+
+[discrete]
+.Converted filterQuery to KQL.(8.11)
+[%collapsible]
+====
+*Details* +
+Converts `filterQuery` to a KQL query string. For more information, refer to ({kibana-pull}161806[#161806]).
+====
+
+[discrete]
+.Moved doc_root.vulnerability.package to doc_root.package (ECS). (8.11)
+[%collapsible]
+====
+*Details* +
+This updates all instances of vulnerability.package to the ECS standard package fieldset. For more information, refer to ({kibana-pull}164651[#164651]).
+====
+
+[discrete]
+.Introduced new summary search capabilities that will cause existing SLOs to stop working. (8.10)
+[%collapsible]
+====
+*Details* +
+New summary search capabilities introduce breaking changes in various places, and we have decided not to handle backward compatibility:
```

Comment:
> we have decided not to handle backward compatibility:

Is there a better way we can word this? I know this was pulled from elsewhere but sounds like bad messaging.

Reply:
```suggestion

```
It's actually repeating the title so we can remove this line